Library/Homebrew/test/language/python/virtualenv_spec.rb

```diff
@@ -3,151 +3,296 @@
 require "language/python"
 require "resource"
 
-RSpec.describe Language::Python::Virtualenv::Virtualenv, :needs_python do
-  subject(:virtualenv) { described_class.new(formula, dir, "python") }
+RSpec.describe Language::Python::Virtualenv, :needs_python do
+  describe "#virtualenv_install_with_resources" do
+    let(:venv) { instance_double(Language::Python::Virtualenv::Virtualenv) }
+    let(:f) do
+      formula "foo" do
+        include Language::Python::Virtualenv # rubocop:disable RSpec/DescribedClass
```

Comment:
Couldn't find a way to get `described_class` to work inside `formula do` so just disabled style check here.

Reply:
```suggestion
        # Couldn't find a way to get described_class to work inside formula do
        # rubocop:disable RSpec/DescribedClass
        include Language::Python::Virtualenv
        # rubocop:enable RSpec/DescribedClass
```